src/components/Dashboard/index.jsx

```diff
@@ -39,6 +42,7 @@ import { user } from '../../utils/prop-types';
     appBarTitle: {
       fontFamily: 'Roboto300',
       flex: 1,
+      color: 'rgba(255, 255, 255, 0.9)',
```

Comment:
Since this is close to white, should this switch based on theme?

Reply:
The app bar title sits behind the blue background which doesn't change on theme toggle, hence the hardcoded value.